symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java

```diff
@@ -63,29 +64,45 @@ protected SymphonyBdk(BdkConfig config, ApiClientFactory apiClientFactory)
     this.config = config;
 
     final AuthenticatorFactory authenticatorFactory = new AuthenticatorFactory(config, apiClientFactory);
-    this.botSession = authenticatorFactory.getBotAuthenticator().authenticateBot();
     this.oboAuthenticator = config.isOboConfigured() ? authenticatorFactory.getOboAuthenticator() : null;
     this.extensionAppAuthenticator =
         config.isOboConfigured() ? authenticatorFactory.getExtensionAppAuthenticator() : null;
 
-    // service init
-    final ServiceFactory serviceFactory = new ServiceFactory(apiClientFactory, this.botSession, config);
-    SessionService sessionService = serviceFactory.getSessionService();
-    this.userService = serviceFactory.getUserService();
-    this.streamService = serviceFactory.getStreamService();
-    this.presenceService = serviceFactory.getPresenceService();
-    this.connectionService = serviceFactory.getConnectionService();
-    this.signalService = serviceFactory.getSignalService();
-    this.applicationService = serviceFactory.getApplicationService();
-    this.healthService = serviceFactory.getHealthService();
-    this.messageService = serviceFactory.getMessageService();
-    this.datafeedService = serviceFactory.getDatafeedService();
-
-    // retrieve bot session info
-    this.botInfo = sessionService.getSession(this.botSession);
-
-    // setup activities
-    this.activityRegistry = new ActivityRegistry(this.botInfo, this.datafeedService::subscribe);
+    if (config.isBotConfigured()) {
+      this.botSession = authenticatorFactory.getBotAuthenticator().authenticateBot();
+      // service init
+      final ServiceFactory serviceFactory = new ServiceFactory(apiClientFactory, this.botSession, config);
+      SessionService sessionService = serviceFactory.getSessionService();
+      this.userService = serviceFactory.getUserService();
```

Comment:
`this.userService = config.isBotConfigured() ? serviceFactory.getUserService() : null`